playwright.config.js

```diff
@@ -5,8 +5,14 @@ const baseURL = "http://127.0.0.1:8080";
 export default defineConfig({
   outputDir: "snapshot_results",
   workers: process.env.CI ? 1 : undefined,
+  expect: {
+    toHaveScreenshot: {
+      threshold: 0.0, // We expect colors to be exact matches.
+    },
+  },
   use: {
     baseURL,
+    viewport: { width: 1920, height: 1080 }, // Sets default viewport to desktop dimensions.
```

Comment:
Copied from https://github.com/Qiskit/qiskit_sphinx_theme/pull/374. We need there for that PR to work. Here, I only did it to make it simpler to generate the screenshot for the footer, that I can reuse a screenshot file from another PR.